src/platform/macos.rs

```diff
@@ -443,13 +443,21 @@ impl MonitorHandleExtMacOS for MonitorHandle {
 
 /// Additional methods on `EventLoopWindowTarget` that are specific to macOS.
 pub trait EventLoopWindowTargetExtMacOS {
+  /// Unhide the application.
+  fn unhide_application(&self);
```

Comment:
can we move this down after `hide_application() 
```suggestion
  /// Show the entire application, see [`EventLoopWindowTargetExtMacOs::hide_application()`].
  fn show_application(&self);
```

Reply:
Sure

Reply:
Couldn't get that doc link to work so just removed it

Reply:
Could you try to fix it ? I think it should be ``[`EventLoopWindowTargetExtMacOs::hide_application`]`` or ``[crate::platform::macos::EventLoopWindowTargetExtMacOs::hide_application]``

Reply:
Tried both of those 😕

Reply:
cc @wusyong

Reply:
If that doesn't work, just use old school markdown link syntax. It's getting too long anyway

Reply:
so ``[`hide_application`](crate::platform::macos::EventLoopWindowTargetExtMacOs::hide_application)`` ?

Reply:
Rustdoc documentation says this about linking, maybe that's the issue:
> Associated items (functions, types, and constants) are supported, but not for blanket trait implementations.

Reply:
if it doesn't work, lets move on, it is not necessary.